core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/transactional/TransactionManager.java

```diff
@@ -2354,15 +2446,15 @@ public Integer call() throws IOException {
                 loopExit = true; 
               } 
               catch (InterruptedException ie) {}
-              catch (CommitUnsuccessfulException cue) {
+              catch (IOException e) {
                  loopExit = true;
-                 LOG.error("Exception at the time of aborting DDL transaction", cue); 
-                 savedCue = cue;
+                 LOG.error("Exception at the time of aborting DDL transaction", e); 
```

Comment:
repeating the same comment to include Exception encountered in DML before processing DDL.